doc/api.apib

```diff
@@ -174,11 +183,342 @@ aglio -i doc/api.apib -o api.html --theme-template triple --theme-variables stre
 + Request (application/json)
 
         {
-            "docker_image": "busybox:latest"
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+        }
+
++ Response 200 (application/json)
+
+        {
+            "enabled": null ,
+            "docker_image": "luigi-canary-dummy:dummy",
+            "commit_sha": "f043333085fa87738ac24f04d64fb58ecc845111",
+            "next_natural_trigger": null
```

Comment:
A PR addressing #117 has been created. So here we can remove `enabled` and `next_natural_trigger` from response.